customizations.md

```diff
@@ -63,6 +63,20 @@ Any system being used to transport images to ECUs needs to modified only to perm
 
 However, in order for an implementation to be Uptane-compliant, no ECU can cause another ECU to install an image without performing either full or partial verification of metadata. This is done in order to prevent attackers from being able to bypass Uptane, and thus execute arbitrary software attacks. Thus, in an Uptane-compliant implementation, an ECU performs either full or partial verification of metadata and images before installing any image, regardless of how the metadata and images were transmitted to the ECU.
 
+
+## Using Uptane with transport security
+Uptane is designed to retain strong security guarantees even in the face of a network attacker.  This includes situations where there either is no transport security or where the transport security is compromised by an attacker.  In this case, Uptane's security loss is that an attacker may disrupt communication between the vehicle and the OEM (e.g., by jamming the signal or dropping packets).  However, malicious packages cannot be installed, mix-and-match attacks are not possible, etc.  This is similar to how a network attacker without key material can cause a TLS connection to fail to connect or disconnect, but cannot compromise the integrity or confidentiality of the connection.
```

Comment:
"Uptane's security loss" sounds a bit odd to me. Can we find a better way to phrase that?

Reply:
I agree with Patrick. I suggest rephrasing the beginning of that third sentence as follows:

"Should this occur, Uptane may not be able to prevent an attacker from disrupting communication between the vehicle and the OEM (e.g...."

In the last sentence in this paragraph, the phrase "key material" strikes me as odd. Can't we just say "...a network attacker who has not compromised a key can cause..."

Reply:
The unfinished bits at the end of my comment can be ignored. I think I based this on an older message and then just forgot to cut out what wasn't needed.
The three suggested edits are in the message,

Lois

Reply:
Okay, I've added the suggestions mentioned.  (Lois, next time please do that instead of listing them only in the comment.)

Can someone outside NYU please review / merge if this is okay?

Reply:
Yes. Will do,